app/forms/waste_exemptions_engine/edit_form.rb

```diff
@@ -23,8 +23,12 @@ class EditForm < BaseForm
 
     # This form has a lot of attributes, so we have to disable the length cop.
     # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
     def initialize(registration)
+      registration.save! unless registration.persisted?
```

Comment:
In todays (yesterdays 😪 ) meeting we said we were not going to touch the edit functionality. But can I assume this is still needed as it fixes something?

Reply:
it does fix something. We said we don't touch it but we need it persisted as it generates too many other fixes. At least this is what I recall 🤔